### PR TITLE
add base images to core

### LINKED ISSUE #3: Move base images to this repo
You could store them in:

    .github/workflows/build_docker_images.yaml
    docker/_viash.yaml
    docker/src/r/config.vsh.yaml
    docker/src/python/config.vsh.yaml
    docker/src/.../config.vsh.yaml

The question is, where do we publish the images to? We could keep them at their current location so we don't need to change the documentation again:

    ghcr.io/openproblems-bio/base_images/r:1.1.0
    ghcr.io/openproblems-bio/base_images/python:1.1.0

We could also move them to `core/base`:

    ghcr.io/openproblems-bio/core/base/r:1.1.0
    ghcr.io/openproblems-bio/core/base/python:1.1.0

OR. Maybe we could also publish the base images to Docker hub instead of ghcr:

We could also move them to Docker hub ( https://hub.docker.com/repositories/openproblems ):

    openproblems/base_r:1.1.0
    openproblems/base_python:1.1.0

That would make it easier for users to type.

@KaiWaldrant WDYT?

### PR DESCRIPTION
**Pull request by KaiWaldrant**
close #3

### CODE REVIEW

File: base_images/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# OpenProblems Base Images v1.0.0
+
+Relocated the base images from base_images repo to core repo.
+
+## MAJOR CHANGES
+
+* Changed container registry from `ghcr.io/openproblems-bio/base_images` to docker hub `openproblems/base_images`. (PR #5")
```
Comment (rcannood) [inline base_images/CHANGELOG.md:7]: Fix formatting ^^

File: base_images/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+<!-- This file is rendered by running 'quarto render README.qmd' -->
+
+Repository for base Docker images for openproblems-bio on Docker-hub.
+
+This repository currently contains the following images:
+
+- [`openproblems/base_images/python:1.0.0`](src/python)
+- [`openproblems/base_images/pytorch_nvidia:1.0.0`](src/pytorch_nvidia)
+- [`openproblems/base_images/r:1.0.0`](src/r)
+- [`openproblems/base_images/tensorflow_nvidia:1.0.0`](src/tensorflow_nvidia)
```
Comment (rcannood) [inline base_images/README.md:13]: Docker hub doesn't support multiple slashes in the docker image name.

I think we'll need to rename it to `openproblems/base_python:1.0.0` and so on.

**Review by rcannood — CHANGES_REQUESTED** [review]

**Comment by KaiWaldrant** [comment]
Close #3

File: base_images/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# OpenProblems Base Images v1.0.0
+
+Relocated the base images from base_images repo to core repo.
+
+## MAJOR CHANGES
+
+* Change container registry from `ghcr.io/openproblems-bio/base_images` to docker hub `openproblems/base_*`. (PR #5)
```
Comment (rcannood) [inline base_images/CHANGELOG.md:7]: * Change container registry from `ghcr.io/openproblems-bio/base_images` to docker hub `openproblems/base_*` (PR #5).

**Review by rcannood — APPROVED** [review]

**Pull request merged**